src/plugins/data/common/search/aggs/utils/to_json.ts

```diff
@@ -35,7 +35,7 @@
  * @internal
  * @deprecated This function will be removed in the next major version.
  */
-export function toAngularJSON(obj: any, pretty?: any): string {
+export function toJSON(obj: any, pretty?: any): string {
```

Comment:
change method signature is same as delete the method.
if change method will not break anything, it should be safely deleted. 

btw, to all osd maintainer, is this change worth to call out in change log, or should we skip change log?

Reply:
A breaking change (including a backward-incompatible change to signature) has a place in the CHANGELOG.